core/src/main/scala-2.13+/cats/instances/all.scala

```diff
@@ -27,6 +27,7 @@ trait AllInstances
     with FutureInstances
     with HashInstances
     with InvariantMonoidalInstances
+    with JavaTimeInstances
```

Comment:
> This is just in kernel? Should this have forwarders so cats imports get this automatically or am I missing them?

@ChristopherDavenport  I think adding this mixin should add the instances to the `core` instances as well.